tests/graphics/artists_test.py

```diff
@@ -10,68 +10,88 @@
 from plopp.data import examples
 
 CASES = {
-    "line-mpl-static": (('2d', 'mpl-static'), pp.plot, (pp.data.data1d(),)),
-    "line-mpl-interactive": (('2d', 'mpl-interactive'), pp.plot, (pp.data.data1d(),)),
+    "line-mpl-static": (('2d', 'mpl-static'), pp.plot, {'obj': pp.data.data1d()}),
```

Comment:
Changes in this file are because `mesh3d` no longer accepts positional arguments